notebooks/MAST/m_dwarf_flares_hst/m_dwarf_flares_hst.ipynb

```diff
@@ -0,0 +1,1412 @@
+{
```

Comment:
<p>These look like good exercises! I think the first one would be good to demonstrate, though. You could drive the point home about the importance of considering proper motion for M dwarfs by taking two images, separted by a lot of time, and plotting them on the RA, Dec plot. We should see the star in two different locations, right? Then an exercise to extend that might be to plot a lot of observations on one plot... or animate them over time? We should be able to see Proxima Centauri move relative to the coordinate axes (if I understand correctly).</p><p> </p><p>But if I actually wanted to stack them on top of each other... that I don't think I have enough information for how to do. I guess somehow I could use the apply_space_motion function to say okay, in this image the star is at RA1, Dec1 and at this later image the star is at RA2, Dec2, and I want those two coordinates to be perfectly aligned. If that's the functionality you were thinking of, it would be good to demonstrate that as well.</p><p> </p><p>And, here's another idea for an exercise, if you think it's useful. The unresolved question in my mind was still, how many observations would I miss if I didn't take proper motion into account? So I searched on MAST for "Proxima Centauri r=0.0014", limited to mission =HST and producttype=Image and calibration level =3. I get 59 results. So that's comparable to the table of length 106 above, right? So you miss nearly half the observations if you don't take into account proper motion! (All the results I get from MAST that way are from 1998, is that weird, or to be expected - those are the closest observations to J2000 epoch? But your length 106 table has observations from other years, is that right?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>